sale_stock_invoice_snln_enhanced/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Sale Stock Invoice Snln Enhanced",
+    "summary": """
+        Enhaces lot/serial number section of sale invoice report.
```

Comment:
Oops, a typo was introduced, it's my fault:
```suggestion
        Enhances lot/serial number section of sale invoice report.
```